test/apps_test.py

```diff
@@ -0,0 +1,84 @@
+"""Test apps.py module"""
+from unittest import TestCase
+from os.path import join, expanduser
+import pytest
+
+from projector_installer.apps import get_app_path, is_path_to_app, parse_version, \
+    get_data_dir_from_script, is_mps_dir
+
+
+class AppsTest(TestCase):
+    """Test apps.py module"""
+
+    app_path = join(expanduser('~'), 'projector/apps/app_path')
+
+    def test_get_app_path(self):
+        """The get_app_path method must return correct full path for the specified app path"""
+        user = expanduser('~')
+        self.assertEqual(get_app_path("app_path"), f'{user}/.projector/apps/app_path')
```

Comment:
Ok, I'll try to find it out. What are high-level paths?

Reply:
Well, I don't know about an exact type in Python, but in Java there is `Path` and it's a high-level object compared to plain `String`. AFAIU, its `equals` method [allows to properly compare](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#equals(java.lang.Object)) relative and absolute paths, without expanding the relative path to an absolute one.

Feel free to postpone this fix to the next PR because it's a very minor thing.